weapons-on-back/cl_weapons-on-back.lua

```diff
@@ -40,27 +41,34 @@ local SETTINGS = {
     }
 }
 
+--- Other Variables ---
 local attached_weapons = {}
+local me = GetPlayerPed(-1)
+local bone = GetPedBoneIndex(GetPlayerPed(-1), boneNumber)
+local toggle = (1,19) --- Change toggle key here. Currently: left alt.
 
-Citizen.CreateThread(function()
-  while true do
-      local me = GetPlayerPed(-1)
-      ---------------------------------------
-      -- attach if player has large weapon --
-      ---------------------------------------
-      for wep_name, wep_hash in pairs(SETTINGS.compatable_weapon_hashes) do
-          if HasPedGotWeapon(me, wep_hash, false) then
-              if not attached_weapons[wep_name] then
+
+if (IsControlJustReleased(toggle)) then --- Toggle here

```

Comment:
This also should be in a while true loop because this is only being executed once when the script starts when we want it to always be listening.

Reply:
Done!